### ISSUE TITLE
On Call resources 403

### ISSUE THREAD

**Issue by alekhrycaiko** [grafana/terraform-provider-grafana#598]
### Terraform Version
* Terraform: 1.1.6
* Terraform Grafana Provider: 1.25
* Grafana: Cloud

### Affected Resource(s)
- Grafana on call resources

### Expected Behavior
Terraform plans succeed.

### Actual Behavior
Terraform is planning with 403's running on fresh tokens against on call resources.

**Comment by alekhrycaiko** [comment]
I'm uncertain @alexweav if this is related to https://github.com/grafana/terraform-provider-grafana/issues/596 but I've been experiencing 403's since Friday using Grafana cloud 

If this is related, any idea when a fix is coming and/or a strategy for temporary resolution here?

**Comment by Konstantinov-Innokentii** [comment]
Hey, I'm working on a fix. For now you can add **oncall_url** param to the provider block:
```
provider "grafana" {
  oncall_access_token = "<YOUR_TOKEN>"
  oncall_url          = "https://oncall-prod-us-central-0.grafana.net/oncall"
}
```

**Comment by julienduchesne** [comment]
Fixed in v1.26.0+
